Cakes-Co/style.css

```
@@ -1 +1,251 @@
+    --generalPadding: 1em;
+    --sectionGap: 2em;
+    --white: rgb(75, 6, 52);
+  }
+  
+  header {
+    height: 40vw;
+    padding: var(--generalPadding);
+    background-color: var(--backgroundColor);
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    color: var(--white);
+  }
+  
+  
+
+
+
+  .logo-img {
+    margin-top: var(--generalPadding);
+    height: 70%;
+    width: auto;
+    border-radius: 50%;
+
+  }
+  .header-area {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: flex-end;
+    text-align: end;
+  }
+  #menu-inpt {
+    display: none;
+  }
+  .menu-lbl {
+    height: 30%;
+    width: auto;
+    position: relative;
+    cursor: pointer;
+  }
+
+  .menu-img {
+    height: 100%;
+  }
+
+  .header-area p {
+    width: 19ch;
+  }
+  
+  .nav-nav {
+    display: none;
+    width: min-content;
+    border-radius: var(--generalPadding);
+    padding: var(--generalPadding);
+    background-color: rgb(231, 119, 162);
+    position: absolute;
+    top: 70px;
+    right: 100%;
+    transition: 0s;
+  }
+  
+  #menu-inpt:checked + .nav-nav {
+    display: block;
+    right: var(--generalPadding);
+  }
+  
+  .nav-ul {
+    width: min-content;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 1em;
+  }
+  .nav-a {
+    border: solid 1px;
+    text-decoration: none;
+    border-radius: 0.5em;
+    background-color: var(--backgroundColor);
+    padding: 0.25em 1em 0.2em;
+    color: var(--white);
+  }
+  
+  main {
+    margin: var(--sectionGap) 0;
+    padding: var(--generalPadding);
+    display: flex;
+    flex-direction: column;
+    gap: var(--sectionGap);
+  }
+  
+  #intro-sctn {
+    display: flex;
+    flex-direction: column;
+    gap: var(--sectionGap);
+  }
+  
+  #images-sctn {
+    display: grid;
+    gap: var(--generalPadding);
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: repeat(2, auto);
+  }
+  
+  footer {
+    height: 40vw;
+    padding: var(--generalPadding);
+    background-color: var(--backgroundColor);
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+  }
+  
+  #nav-footer {
+    display: flex;
+    flex-direction: column;
+    gap: calc(var(--generalPadding) / 2);
+  }
+  #nav-footer a {
+    text-decoration: none;
+    color: var(--white);
+    font-size: 1em;
+    font-weight: 400;
+  }
+  
+  footer h6 {
+    align-self: center;
+    color: var(--white);
+  }
+  
+  @media screen and (min-width: 540px) {
+    body {
+      font-size: 2vw;
+    }
+    header {
+      height: 10vw;
+    }
+    .header-area {
+      justify-content: flex-end;
+    }
+  
+    .nav-nav {
+      margin: var(--generalPadding) var(--generalPadding) 0;
+      display: flex;
+      width: auto;
+      border-radius: 0;
+      position: static;
+      justify-content: flex-end;
+    }
+  
```

consider using something like this 
```suggestion
  nav ul
```